opendevin/runtime/od_runtime_client/mock_test/client.py

```diff
@@ -0,0 +1,61 @@
+# client.py, this file is used in development to test the runtime client. It is not used in production.
+import asyncio
+import websockets
+
+# Function for sending commands to the server in EventStreamRuntime
+class EventStreamRuntime:
+    uri = 'ws://localhost:8080'
+
+    def __init__(self):
+        self.websocket = None
+
+    def _init_websocket(self):
+        self.websocket = None
+        # self.loop = asyncio.new_event_loop()
```

Comment:
Remove commented out lines

Reply:
Want to keep it. Because as mentioned in description, this is actually a potention optimization point. Will add a TODO in the comments to mention it.